tests/format/js/tagged-template-expression-ts/format.test.js

```diff
@@ -0,0 +1 @@
+runFormatTest(import.meta, ["typescript"]);
```

Comment:
This test need move to the typescript language directory.

Reply:
Done!